charts/rancher-monitoring/v0.3.0/questions.yml

```diff
@@ -0,0 +1 @@
+rancher_min_version: 2.6.0-rc1
```

Comment:
**Question:** Should the value be `2.6.0-0` in case we want this chart showing in Rancher versions like 2.6.0-alpha or so?

Reply:
Yes I think that makes sense.